src/benchmarklib/cli_config_parser.cpp

```diff
@@ -137,6 +138,29 @@ BenchmarkConfig CLIConfigParser::parse_cli_options(const cxxopts::ParseResult& p
     std::cout << "- Not tracking SQL metrics" << std::endl;
   }
 
+  const auto enable_temporary_memory_tracking = parse_result["memory_tracking"].as<bool>();
+  if (enable_temporary_memory_tracking) {
+    std::cout << "- Tracking temporary memory usage."
+              << " Caution: Temporary memory usage tracking is not implemented for all operators" << std::endl;
+  } else {
+    std::cout << "- Not tracking temporary memory usage" << std::endl;
+  }
+
+  std::optional<std::string> memory_tracking_output_file_path;
+  const auto memory_tracking_output_file_string = parse_result["memory_tracking_output"].as<std::string>();
+  if (!memory_tracking_output_file_string.empty()) {
+    if (enable_temporary_memory_tracking) {
+      memory_tracking_output_file_path = memory_tracking_output_file_string;
+      std::cout << "- Writing temporary memory usage stats results to '"
+      << memory_tracking_output_file_path.value() << "'" << std::endl;
```

Comment:
If you want to reduce the line length, you could just create a new line for `<< memory_tracking_output_file_path.value() << "'" << std::endl;` and add a `std::cout` in front

Reply:
For readability reasons, or what would be the improvement here?
I'm also not really happy with the long print-statements.

Reply:
yes, this would lead to shorter lines